utilities/filter_molecule_sets/filter_molecule_sets.py

```diff
@@ -65,18 +94,18 @@ def filter_molecules(input_molstream, output_molstream, allow_repeats = False,
             smile_count += 1
 
         if new_smile or allow_repeats:
-            keep = keep_molecule(mol_copy, max_heavy_atoms, remove_smirks, max_metals)
+            keep = keep_molecule(mol_copy, max_heavy_atoms, remove_smirks, max_metals, elements, check_type)
             if keep:
                 smiles.append(smi)
                 oechem.OEWriteMolecule(output_molstream, mol_copy)
                 saved += 1
-
         errs.clear()
 
-    print("%i molecules in %s" % (count, input_file))
+    print("%i molecules in %s" % (count, input_molstream))
```

Comment:
Does printing the name of the input and output molecule streams actually do anything useful? Naively I would have guessed it would just print some sort of "oemolostream" python name.